engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/compile/ProcessValidatorSpec.scala

```diff
@@ -155,7 +156,27 @@ class ProcessValidatorSpec extends FunSuite with Matchers with Inside {
     val compilationResult = validate(testProcess, baseDefinition)
 
     compilationResult.result should matchPattern {
-      case Valid(_) =>
+      case Invalid(NonEmptyList(ExpressionParseError(_, _, _, _), _)) =>
```

Comment:
you can be more precise checking validation message

Reply:
you're right, changed it, thanks